cpp-ch/local-engine/Parser/SerializedPlanParser.h

```diff
@@ -177,7 +177,8 @@ static const std::map<std::string, std::string> SCALAR_FUNCTIONS
        {"array", "array"},
        {"shuffle", "arrayShuffle"},
        {"range", "range"}, /// dummy mapping
-        {"flatten", "sparkArrayFlatten"},
+       {"flatten", "sparkArrayFlatten"},
+       {"array_join", "sparkArrayJoin"},
```

Comment:
Please don't add new functions by this map

Reply:
Use the `FunctionParser`

Reply:
array_join not has function to map in clickhouse